nekoyume/Assets/_Scripts/UI/Module/Button/AcquisitionPlaceButton.cs

```diff
@@ -128,11 +127,13 @@ private void EnableSettingByPlaceType(PlaceType type, Model model)
             switch (type)
             {
                 case PlaceType.Stage:
-                    var successToGetUnlockedWorld = States.Instance.CurrentAvatarState.worldInformation
+                    var sharedViewModel = Widget.Find<WorldMap>().SharedViewModel;
+                    var successToGetUnlockedWorld = sharedViewModel.WorldInformation
```

Comment:
Why you change the ref of the `WorldInformation` model from `States` to `WorldMap` UI?

Reply:
It needs `WorldInformation` and `List<int> UnlockedWorldIds`.
States.Instance has only `WorldInformation`, but `WolrdMap.SharedViewModel` has `WorldInformation`(It from ReactiveAvatarState) with `List<int> UnlockedWorldIds`. Therefore I changed it.

Reply:
OK. I have some idea and I wrote some code to caching data which is combined or edited with chain data and other data to using in client side. Let's talk about this in [the discord channel](https://discord.com/channels/928926944937013338/970554251308830750).